docs/developer_guide/registries.md

```diff
@@ -0,0 +1,66 @@
+# Using private Docker registries
+
+To get deployed, your application may need to pull a Docker image stored in a
+private Docker registry. Fortunately, Arnold supports this common scenario.
+
+## Create OKD Secret that stores Docker registry credentials
+
+First, you will need to declare the credentials to login to this private
+registry in a dedicated Ansible vault with the following expected path:
+
+```
+group_vars/customer/{{ customer }}/{{ env_type }}/secrets/registries.vault.yml
+```
+
+This vault should contain a list of Docker registries with the following
+format:
+
+```yaml
+# Docker private registries
+registries:
+  - name: foo
+    server: https://foo.com
+    username: foo
+    password: pass
+    email: foo@example.com
```

Comment:
The email is required ? Why do we need it ?

Reply:
It is required in the current implementation, but it's not required for the docker registry authentication. I can make it optional.

Reply:
+1 for removing it. I don't see the point of it. Your email is already configured in your registries account...

Reply:
Ok, then completely removing it is the easiest way to go :smile:

Reply:
(even though I still think it's more explicit to have it in our configuration)

Reply:
Fixed!